daemon/networkdriver/bridge/driver.go

```diff
@@ -82,6 +91,74 @@ var (
 	currentInterfaces = ifaces{c: make(map[string]*networkInterface)}
 )
 
+// Get kernel param path saying whether IPv${ipVer} traffic is being forwarded
+// on particular interface. Interface may be specified for IPv6 only. If
+// `iface` is empty, `default` will be assumed, which represents default value
+// for new interfaces.
+func getForwardingKernelParam(ipVer ipVersion, iface string) (res string) {
+	switch ipVer {
+	case ipv4:
+		res = "/proc/sys/net/ipv4/ip_forward"
+	case ipv6:
+		if iface == "" {
+			iface = "default"
+		}
+		res = fmt.Sprintf("/proc/sys/net/ipv6/conf/%s/forwarding", iface)
+	}
+	return
+}
+
+// Get kernel param path saying whether bridged IPv${ipVer} traffic shall be
+// passed to ip${ipVer}tables' chains.
+func getBridgeNFKernelParam(ipVer ipVersion) (res string) {
+	switch ipVer {
+	case ipv4:
+		res = "/proc/sys/net/bridge/bridge-nf-call-iptables"
+	case ipv6:
+		res = "/proc/sys/net/bridge/bridge-nf-call-ip6tables"
+	}
+	return
+}
+
+func getKernelBoolParam(path string) (bool, error) {
+	enabled := false
+	line, err := ioutil.ReadFile(path)
+	if err != nil {
+		err = fmt.Errorf("Failed to read kernel parameter \"%s\": %v", path, err)
+	} else if len(line) > 0 {
+		enabled = line[0] == '1'
+	}
+	return enabled, err
+}
+
+func isPacketForwardingEnabled(ipVer ipVersion, iface string) (bool, error) {
+	var (
```

Comment:
You don't need vars mostly. You can just return immediately when you get result.